### PR TITLE
Deteleted outdatat docs, updated README.md

### PR DESCRIPTION
**Pull request by ZbyszekMM**
With the arrival of completely new documentation set it is time to clean up outdated stuff.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
+Nussknacker lets you design, deploy and monitor streaming data processing scenarios using GUI. You can run your scenarios on various engines. 
 
-Nussknacker started as Apache Flink "self-service" process authoring tool. We still leverage the power, performance and reliability of Apache Flink to make your streaming processes fast and accurate. 
+Nussknacker started as Apache Flink "self-service" processing scenarios authoring tool. We still leverage the power, performance and reliability of Apache Flink to make processing of stream data fast and accurate. 
 
-But you are not limited to Flink. The request-response engine lets you create processes exposing REST API.
+But you are not limited to Flink. The request-response engine lets you create processing scenarios exposing REST API.
 
 See:
 * [Quickstart](Quickstart.md) to have a look around
```
Comment (mproch) [inline docs/README.md:10]: this also should be changed?

File: docs/README.md [hunk 2]
```diff
@@ -1,23 +1,17 @@
 ![logo](img/nussknacker_small.jpg)
 
-Nussknacker lets you design, deploy and monitor processes using GUI. You can run your processes on various engines. 
+Nussknacker lets you design, deploy and monitor streaming data processing scenarios using GUI. You can run your scenarios on various engines.
```
Comment (mproch) [inline docs/README.md:3]: Let's remove streaming from here

File: docs/README.md (hunk 1 above)
Comment (ZbyszekMM) [reply docs/README.md:10]: updated link to demo version

Comment (ZbyszekMM) [reply docs/README.md:10]: updated link to demo version

File: docs/README.md (hunk 2 above)
Comment (ZbyszekMM) [reply docs/README.md:3]: removed

**Review by dswiecki — APPROVED** [review]

**Pull request merged**